packages/atlas/src/components/_inputs/Slider/RatioSlider.tsx

```diff
@@ -0,0 +1,78 @@
+import { ChangeEventHandler, forwardRef, useMemo, useState } from 'react'
+
+import { sizes } from '@/styles'
+
+import { Range, Track, Wrapper } from './RatioSlider.styles'
+
+type Props = {
```

Comment:
For the sake of code unity let's go with the `{component name}Props` naming convention for props, plus we can export it and import it in the story to not duplicate the code, plus if smth changes on the component we would receive some errors on story to fix it.